eq_translations/survey_schema.py

```diff
@@ -162,8 +117,13 @@ def catalog(self):
                 continue
 
             message_id = get_message_id(translatable_item.value)
+            auto_comments = [translatable_item.description]
```

Comment:
Any idea is auto comments and user comments are treated differently in crowdin and whether they have any certain significance?:
```pot
# user_comment
#. auto_comment
msgctxt "a ctx"
```

Reply:
I've just checked and they're presented as they are in the pot file, and both are editable unfortunately. Gettext itself defines a number of different auto comments (https://www.gnu.org/software/gettext/manual/gettext.html#PO-Files), referring to the `#.` as extracted comments, which I think fits with our usage.